tests/test_ptable.py

```diff
@@ -318,7 +318,7 @@ def test_ptable_heatmap_plotly_cscale_range(
     fig = ptable_heatmap_plotly(df_ptable.density, cscale_range=cscale_range)
     trace = fig.data[0]
     assert "colorbar" in trace
-    # check for correct colorbar range
+    # check for correct color bar range
```

Comment:
I think we agreed on using `colorbar` instead of `color bar` here...Maybe your spelling checking plugin not updated? @janosh

Reply:
yes, the spell checker fixes it. i don't think it makes sense to override the spell checker here since it is actually correct that this is not a word

Reply:
Maybe you want to consider add it to the workplace settings? In VScode it's just `right click`-`spelling`-`Add words to workplace settings`. I think it make sense to consider some package-specific terminologies like this, just like `CONTCAR` itself is not a word, but I still add it to VASP-related project workplaces to avoid being warned anyway.